File: ghislain-one-app/src/app/business/artist.ts
import {Injectable, OnInit} from '@angular/core';
import { Album } from './album';
declare var $:any;
 

@Injectable()
export class Artist {
    Id:number = 0;
    ArtistName:string = null;
    Description:string = null;
    ImageUrl:string = null;
    AmazonUrl:string = null;
    AlbumCount:number = 0;
    Albums:Album[] = [];
}
 

File: ZoeHome/src/ZoeHome.Domain/Entities/Artist.cs
namespace ZoeHome.Domain.Entities
{
    public class Artist
    {
        public string AmazonUrl { get; set; }
        public string ArtistName { get; set; }
        public string Description { get; set; }
        public int Id { get; set; }
        public string ImageUrl { get; set; }
        //public List<Album> Albums { get; set; }
    }
}